homeassistant/components/dominos.py

```diff
@@ -77,11 +77,12 @@ def setup(hass, config):
     if conf.get(ATTR_SHOW_MENU):
         hass.http.register_view(DominosProductListView(dominos))
 
-    for order_info in conf.get(ATTR_ORDERS):
-        order = DominosOrder(order_info, dominos)
-        entities.append(order)
+    if conf.get(ATTR_ORDERS) is not None:
```

Comment:
I'd rather default to an empty list in the config schema and add a check below if `entities` is empty before adding them.